scripts/install.sh

```diff
@@ -53,15 +54,15 @@ chown oracle:oinstall -R /opt
 echo 'INSTALLER: Oracle directories created'
 
 # set environment variables
-echo "export ORACLE_BASE=/opt/oracle" >> /home/oracle/.bash_profile \
- && echo "export ORACLE_HOME=/opt/oracle/product/12.1.0.2/dbhome_1" >> /home/oracle/.bash_profile \
- && echo "export ORACLE_SID=orcl" >> /home/oracle/.bash_profile \
- && echo "export PATH=\$PATH:\$ORACLE_HOME/bin" >> /home/oracle/.bash_profile
+echo "export ORACLE_BASE=/opt/oracle" >> /home/oracle/.bashrc \
+ && echo "export ORACLE_HOME=/opt/oracle/product/12.1.0.2/dbhome_1" >> /home/oracle/.bashrc \
+ && echo "export ORACLE_SID=orcl" >> /home/oracle/.bashrc \
+ && echo "export PATH=\$PATH:\$ORACLE_HOME/bin" >> /home/oracle/.bashrc
 
 echo 'INSTALLER: Environment variables set'
 
 # install Oracle
-su -l oracle -c "yes | /vagrant/database/runInstaller -silent -showProgress -ignorePrereq -waitforcompletion -responseFile /vagrant/db_install.rsp"
+su -l oracle -c "yes | /vagrant/database/runInstaller -silent -showProgress -ignorePrereq -waitforcompletion -responseFile /vagrant/ora-responce/db_install.rsp"
```

Comment:
Looks great. The only thing is the spelling of response, with an s. Then I'll merge this.

Cheers!